pyLDAvis/js/ldavis.v3.0.0.js

```diff
@@ -178,7 +182,7 @@ var LDAvis = function(to_select, data_or_file_name, color1, color2) {
                 if (termElem !== undefined) term_off(termElem);
                 vis_state.term = "";
                 var value_old = document.getElementById(topicID).value;
-                var value_new = Math.min(K, +value_old + 1).toFixed(0);
+                var value_new = Math.min(K - 1 + startIndex, +value_old + 1).toFixed(0);
```

Comment:
`K - 1 + startIndex` -> `K + startIndex - 1`

Just to be consistent everywhere

Reply:
added commit 0a965059a9fca360e4e4ea212d88fd8657327bed